recipes/ervdetective/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# build 
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
You could move this to meta.yaml > `build > script`

Reply:
Thanks for the reminder. Could you help me with that? Because each modification requires re-entering the test and waiting for the review. If encounter a new merge, and  need to re-run the test.